R/geom-function.R

```diff
@@ -32,6 +32,8 @@
 #'
 #' base + stat_function(fun = dnorm, geom = "point", n = 20)
 #'
+#' base + stat_function(fun = dnorm, geom = "density", color = "blue", fill = "blue", alpha = 0.5)
```

Comment:
any reason why not using `geom = "polygon"` - it seems more general...

Reply:
No, no specific reason. `geom = "density"` seemed a little bit more intuitive since the example is about plotting a probability density function. But I have no strong feelings on this.

Reply:
I think we should change it

Reply:
Done